texparser/tar_extractor.py

```diff
@@ -13,28 +15,34 @@ def extract_folder(self, folder, func):
         filenames = os.listdir(folder)
         for filename in filenames:
             try:
-                processes.append(func(folder + "/" + filename))
-            except:
+                path = Path(folder) / filename
+                filename, file_extension = os.path.splitext(path)
+                processes.append(func(filename + file_extension))
+            except FileExistsError:
+                sys.stderr.write("Error message: File already exists. \n")
                 pass  # How should we handle already existing folders?
```

Comment:
remove comment or ammend it plox :)

Reply:
comment is deleted

Reply:
commit:#e96f423d1bf6ed19836192c79e08eb31510dfaac

Reply:
commit: #e96f423d1bf6ed19836192c79e08eb31510dfaac